dev/temp_norm_annot_patch_wf.cwl

```diff
@@ -0,0 +1,77 @@
+cwlVersion: v1.0
+class: Workflow
+id: kfdrc_norm_annot_wf
+doc: "This is a temp wf to normalize and annotate a PASS vcf"
+requirements:
+  - class: ScatterFeatureRequirement
+  - class: MultipleInputFeatureRequirement
+  - class: SubworkflowFeatureRequirement
+
+inputs:
+  indexed_reference_fasta: {type: File, secondaryFiles: [.fai, ^.dict]}
```

Comment:
Could I ask what secondary file the `^.dict` refers to here? Also, here the strings are not in quotes like '.tbi' below; does this have any significance?

Reply:
It's the sequence dictionary:
https://gatk.broadinstitute.org/hc/en-us/articles/360036729911-CreateSequenceDictionary-Picard-
The `^` notation is explained here: https://www.commonwl.org/v1.0/CommandLineTool.html#CommandInputParameter
`If string begins with one or more caret ^ characters, for each caret, remove the last file extension from the path (the last period . and all following characters).`

As far as I know there's no significance to adding quotes. Although sometimes the validator trips up on certain things and you need to add them like for valueFrom statements.